lighthouse-plugin-publisher-ads/audits/loads-ad-tag-over-https.js

```diff
@@ -69,35 +72,35 @@ class LoadsGptOverHttps extends Audit {
       return auditNotApplicable.NoRecords;
     }
 
-    const gptRequests = networkRecords
-        .filter((record) => isGptTag(new URL(record.url)));
+    const adRequests = networkRecords
+        .filter((record) => isAdTag(new URL(record.url)));
 
-    const secureGptRequests = gptRequests.filter((request) => request.isSecure);
+    const secureAdRequests = adRequests.filter((request) => request.isSecure);
 
     /** @type {LH.Audit.Details.DebugData} */
     const details = {
       type: 'debugdata',
-      numGptHttpReqs: gptRequests.length - secureGptRequests.length,
-      numGptHttpsReqs: secureGptRequests.length,
+      numAdTagHttpReqs: adRequests.length - secureAdRequests.length,
+      numAdTagHttpsReqs: secureAdRequests.length,
     };
 
-    if (!gptRequests.length) {
-      const returnVal = auditNotApplicable.NoGpt;
+    if (!adRequests.length) {
+      const returnVal = auditNotApplicable.NoTag;
       returnVal.details = details;
       return returnVal;
     }
 
     return {
-      numericValue: details.numGptHttpReqs,
-      score: details.numGptHttpReqs ? 0 : 1,
-      displayValue: details.numGptHttpReqs ?
+      numericValue: details.numAdTagHttpReqs,
+      score: details.numAdTagHttpReqs ? 0 : 1,
+      displayValue: details.numAdTagHttpReqs ?
```

Comment:
Can you add a TODO to include a details to note which scripts are loaded over http?

I think we can shorten the audit description after we add the details table.

Reply:
Done